### ISSUE TITLE
Add a fontstyle option similar to current fontsize option

### ISSUE THREAD

**Issue by jlstevens** [ioam/holoviews#526]
I would recommend we always support the options supported by bokeh (``['normal'  | 'italic' | bold']``) and in matplotlib we can support the additional weight options `` [ 'heavy' | 'light' | 'ultrabold' | 'ultralight']``.

The only typical option I can think of that isn't covered by this is underlined text...

**Comment by philippjfr** [comment]
matplotlib also has a separate argument for the style separate from the fontweight setting listed above:

style or fontstyle	[ 'normal' | 'italic' | 'oblique']

**Comment by jlstevens** [comment]
Issue #2008 above (now closed) was about color which seems like a much bigger space of possibilities than what has been discussed here so far.

**Comment by philippjfr** [comment]
True, I didn't check this issue sufficiently beforehand but I don't think the limited set of options outlined above here cover the full extent of things you might want to customize anyway. There should be ways of controlling the alpha, horizontal/vertical alignment, the actual font and the color.
